web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedbacklasthome.html

```diff
@@ -7,8 +7,15 @@
         <span data-ng-hide="row.published"> -
           <strong data-translate="">GUFwaitingForApproval</strong>
         </span>
+
+        <a class="btn btn-xs btn-link pull-right"
+           data-ng-if="allowDelete"
+           data-ng-click="deleteC(row.uuid)"
+           title="{{'GUFdeleteUserFeedback' | translate}}">
+          <i class="fa fa-fw fa-times text-danger"></i>&nbsp;
```

Comment:
When possible please remove the `&nbsp;`

Reply:
@fxprunayre somehow I still see it, sorry.

Reply:
Fixed. Also aligned with merge and prettier formatting. Should be good now.